admin/src/js/controllers/edit-user.js

```diff
@@ -86,25 +87,41 @@ angular
       return $scope.model.facility_id;
     };
 
+    const getOidcUsername = function () {
+      if (
+        !$scope.model
+        || !$scope.model._id
+        || !$scope.model.name
+        || !$scope.model.oidc_login
+      ) {
+        return Promise.resolve(undefined);
+      }
+
+      return $http
+        .get(`/api/v2/users/${$scope.model.name}`)
+        .then(({ data: { oidc_username } }) => oidc_username);
+    };
+
     const determineEditUserModel = function() {
       // Edit a user that's not the current user.
       // $scope.model is the user object passed in by controller creating the Modal.
       // If $scope.model === {}, we're creating a new user.
-      return Settings()
-        .then(settings => {
+      return $q.all([Settings(), getOidcUsername()])
+        .then(([settings, oidcUsername]) => {
           $scope.permissions = settings.permissions;
           $scope.roles = settings.roles;
           $scope.allowTokenLogin = allowTokenLogin(settings);
+          $scope.allowSSOLogin = allowSSOLogin(settings);
           if (!$scope.model) {
             return $q.resolve({});
           }
 
           // Start with the password masked.
-          $scope.model.passwordFieldType  = 'password';
+          $scope.model.passwordFieldType = 'password';
 
           $scope.model.showPasswordIcon = SHOW_PASSWORD_ICON;
           $scope.model.hidePasswordIcon = HIDE_PASSWORD_ICON;
-
+          $scope.model.oidc_username = oidcUsername;
```

Comment:
The other `$scope.model` properties here are coming from the `user-settings` document for the user being edited.  The `oidc_username` is stored on the user's doc in the `_users` db and not on the `user-settings` doc in the `medic` db.  So, to get the current `oidc_username` value we have to retrieve the user's data above via the `/api/v2/users` api.